src/startup/routes.jsx

```diff
@@ -9,4 +10,8 @@ export const routes = [{
 {
   path: '/',
   Component: HomePage,
+}, {
+  path: '/user',
```

Comment:
Let's respect our convention and have the `/` route be the last in this array.

Reply:
Done